### ISSUE TITLE
Download script failed on a few sources

### ISSUE THREAD

**Issue by julie-sullivan** [intermine/intermine#988]
1. Reactome failed. They added underscores to their file names. Fixed by bcfb2715f32a2d9355dd93fffb0612190f26034f

2. HPO download failed;

```
Apr 23 14:47:11 [INFO  ] Starting data download for Human Phenotype Ontology
Apr 23 14:47:11 [DEBUG ] Downloading http://compbio.charite.de/hudson/job/hpo/lastStableBuild/artifact/ontology/release//hp.obo to /tmp/0Nx2g8Lqmi
Apr 23 14:47:11 [EMERGENCY] 404 Not Found at line 110
Apr 23 14:47:11 [DEBUG ] Error fetching Human Phenotype Ontology () hp.obo => /micklem/data/hpo/2015-04-23/hp.obo
Apr 23 14:47:11 [DEBUG ] 404 Not Found at /usr/local/lib/perl/5.10.1/Moose/Meta/Method/Delegation.pm line 110
Apr 23 14:47:11 [DEBUG ]  at /usr/local/share/perl/5.10.1/Ouch.pm line 28
        Ouch::ouch('404 Not Found at /usr/local/lib/perl/5.10.1/Moose/Meta/Method...') called at /usr/local/share/perl/5.10.1/Ouch.pm line 33
        Ouch::throw('404 Not Found at /usr/local/lib/perl/5.10.1/Moose/Meta/Method...') called at bin/../lib/DataDownloader/Source/ABC.pm line 112
        DataDownloader::Source::ABC::fetch_all_data('DataDownloader::Source::HPO=HASH(0x4f44d40)') called at bin/../lib/DataDownloader/Role/Source.pm line 180
        DataDownloader::Role::Source::__ANON__() called at /usr/local/share/perl/5.10.1/Ouch.pm line 22
        eval {...} called at /usr/local/share/perl/5.10.1/Ouch.pm line 22
        Ouch::try('CODE(0x4983bd0)') called at bin/../lib/DataDownloader/Role/Source.pm line 183
        DataDownloader::Role::Source::get_data('DataDownloader::Source::HPO=HASH(0x4f44d40)') called at bin/download_data line 75
        eval {...} called at bin/download_data line 73
Apr 23 14:47:11 [ERROR ] Error downloading data for Human Phenotype Ontology:  at bin/../lib/DataDownloader/Source/ABC.pm line 112
Apr 23 14:47:11 [DEBUG ] Removing /micklem/data/hpo/2015-04-23
```
3. Remove ensembl compara

4. The script didn't log its status in /data/download_logs so I had to look at the console. I think I disabled logging for reasons but I don't remember what those reasons were. I am certain it was a good reason.

Tell @rachellyne

**Comment by julie-sullivan** [comment]
HPO fixed by 6bbbd635ac6b2b1991d61bf66d8689a0d51f3c66

**Comment by julie-sullivan** [comment]
Ensembl fixed by 2f0a7c7542c7721d6f28055dd2f60836df6a31b5
